pkg/cli/cli.go

```diff
@@ -1358,7 +1358,13 @@ func runConnectionWebSocketAgentCommand(flags *ConnectFlags, c *command, args []
 		},
 	}
 	device := args[0]
-	wsUrl := fmt.Sprintf("%s/devices/%s/adb", wsRoot, device)
+	var wsUrlBuilder strings.Builder
```

Comment:
let's isolate this logic into a new function `wsURL()`.

Reply:
Extracted the logic into the function wsURL(...).